README.md

```diff
@@ -72,12 +72,11 @@ And the corresponding "date" function that returns a date for
 each data point.
 
 ```js
-var eventDropsChart = d3.chart.eventDrops();
+var eventDropsChart = d3.chart.eventDrops()
+                              .date(d => d.date);
```

Comment:
Can you fix indentation? 

I tried to tell @sedy-bot to do it, but it seems it is not in a correct mood today (ping @Kmaschta). :p